Makefile

```diff
@@ -153,6 +159,65 @@ release: ## Build `skywire-visor`, `skywire-cli`, `hypervisor` and apps without
 	${OPTS} go build ${BUILD_OPTS} -o ./apps/vpn-server ./cmd/apps/vpn-server
 	${OPTS} go build ${BUILD_OPTS} -o ./apps/vpn-client ./cmd/apps/vpn-client
 
+package-amd64: ## Build the debian package. USE ROOT FOR THIS.
```

Comment:
For the following targets, can we put the majority of the logic here in a separate script under the `/scripts` directory? Thank you

Reply:
Yes, it's preferable for this to all be in a script. I did it this way as a demonstration.

Reply:
done

Reply:
Would you mind moving the script into /scripts?